go/vt/binlog/grpcbinlogplayer/player.go

```diff
@@ -27,11 +27,20 @@ import (
 	"github.com/youtube/vitess/go/vt/binlog/binlogplayer"
 	"github.com/youtube/vitess/go/vt/grpcclient"
 
+	"flag"
+
 	binlogdatapb "github.com/youtube/vitess/go/vt/proto/binlogdata"
 	binlogservicepb "github.com/youtube/vitess/go/vt/proto/binlogservice"
 	topodatapb "github.com/youtube/vitess/go/vt/proto/topodata"
 )
 
+var (
+	cert = flag.String("binlog_player_grpc_cert", "", "the cert to use to connect")
```

Comment:
is the the binlog player cert or just the tablet cert?

Reply:
client side cert